2018-11/index.md

```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
+ body: <p><b>Eureka!</b> Εύρηκα, in Greek, is an exclamation we often use to celebrate the success of a discovery or invention. Its first use is attributed to ancient Greek mathematician and inventor Archimedes.</p><p>One of the greatest feelings an inventor can experience is that moment when something works, when the code is de-bugged, when the pieces come together. Enabling success, supporting invention, and encouraging entrepreneurship is the foundation of the BeagleBoard.org community. Open Source hardware and software is by its very nature a place for creativity to blossom into successful invention.</p><p>We’ve recently attended <a href="https://beagleboard.org/blog/2018-10-18-empowering-innovative-education-celebrating-community-maker-faire-nyc">Maker Faire New York</a>, a great place to interact with creative inventors and educators of all ages, backgrounds, and experience levels. In this month’s dish, learn more about community members who have taken their ideas to great heights, like the <a href="https://beagleboard.org/p/bigjosh/three-story-tall-clock-final-entry-5ab263">three story tall clock</a>, or around the world, like the successful <a href="https://beagleboard.org/blog/2018-10-22-poursteady">coffee equipment company</a> who got their start at a Maker Faire. We celebrate these, and also that spark when a student shouts “Eureka!” after lighting their first LED.<br /><p>Educators, please remember to apply by November 29, 2018 for a free donation of up to 30 <a href="https://beagleboard.org/pocketbeagles-for-your-classroom">PocketBeagles® for your classroom.</a> You can participate by submitting a project. Email me at <a href="christi@beagleboard.org">christi@beagleboard.org</a> for more information.  We look forward to helping you to success.<br>&mdash;<strong>Christine Long</strong>, <em>Executive Director</em></p>
 events:
- - title: World MakerFaire Education Forum, New York City
-   date: September 21, 2018 
-   link: https://beagleboard.org/blog/2018-09-05-join-beagleboard-org-at-world-maker-faire-in-new-york-sept-21-23
- - title: BeagleBoard.org 10th Anniversary &num;BringAHack Event, New York City
-   date: September 21, 2018 
-   link: https://www.eventbrite.com/e/bringahack-new-york-with-beagleboardorg-before-maker-faire-tickets-49521195228
- - title: World MakerFaire, New York City
-   date: September 22-23, 2018
-   link: https://beagleboard.org/blog/2018-09-05-join-beagleboard-org-at-world-maker-faire-in-new-york-sept-21-23
+ - title: Embedded World 2019 Exhibition & Conference, Nuremberg, Germany
+   date: February 26-28, 2019 
+   link: https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-38827067/beagleboard-org-foundation
+ - title: Linux Jumpstart with PocketBeagle Hands-On Workshop at Embedded World
```

Comment:
Did you mean to use the phrase "Jumpstart"? I don't think I saw that before. That has typically been used by Bill Gatliff for his courses.

Reply:
Official name is "Getting Started with PocketBeagle from BeagleBoard.org - A Hands-On Coding with Embedded Linux Workshop"

Reply:
On the name - the Embedded World coordinator asked if they could shorten the name of the workshop and made that recommendation.   " Linux Jumpstart with PocketBeagle from BeagleBoard.org"  see this page and scroll down.  https://www.embedded-world.eu/program.html